lib/offsite_payments/integrations/migs.rb

```diff
@@ -0,0 +1,231 @@
+module OffsitePayments
+  module Integrations
+    module Migs
+      API_VERSION = 1
+
+      def self.return(query_string, options = {})
+        Return.new(query_string, options)
+      end
+
+      def self.base_url
+        'https://migs.mastercard.com.au/vpcpay'
+      end
+
+      class Helper < OffsitePayments::Helper
+        def initialize(order, account, options = {})
+          @credentials = { login: account, password: options.fetch(:credential2) }
```

Comment:
Further looking at this I think I could change references to 2 to 1, credential 2 would be if there were further credentials after 1. I'll hold off on making change till we pair, just noting it.